### ISSUE TITLE
MASK RCNN train coco dataset, test on CodaLab for COCO test-dev, seg APL is only 48, which is much lower than 52.4

### ISSUE THREAD

**Issue by ym666888** [open-mmlab/mmdetection#10950]
MASK RCNN train coco dataset, use resnet101 weights, test on CodaLab for COCO test-dev, for split mask test other metrics can reach the accuracy of the paper, but get the value of APL is only 48, which is much lower than 52.4 given by the maskrcnn paper.  thanks for your help

**Comment by ym666888** [comment]
All parameters remain unchanged, as only the learning rate and batch size are changed
